Automattic/pocket-casts-android
# Add fixing downloads option

## Description

This aligns database with the downloads directory. It might help in cases were database got somehow out of sync with the downloaded files.

Internal ref: p1715263428170299-slack-C02A333D8LQ

## Testing Instructions

1. Open the app.
2. Download some episodes.
3. Go to `Profile`/`Downloads` and confirm that you see the episodes.
4. Go back to `Profile`.
5. Execute this query `UPDATE podcast_episodes SET downloaded_file_path = NULL, episode_status = 0`.
6. Go to `Profile`/`Downloads` and confirm that you don't see any episodes.
7. Go to `Settings`/`Storage & data use`.
8. Tap on `Fix downloads`.
9. Wait until process finishes.
10. Go to `Profile`/`Downloads` and confirm that you see the episodes.

## Screenshots or Screencast 

https://github.com/Automattic/pocket-casts-android/assets/30936061/9cb4b167-adb5-4427-8180-77ffd1b1a712

## Checklist
 
#### I have tested any UI changes...
